src/providers/formattingProvider.ts

```diff
@@ -57,6 +57,11 @@ export class FormatProvider implements vscode.DocumentFormattingEditProvider {
         let oneCommandCode = false;
         let blockComment = false;
         let atTopLevel = true;
+        /** Save important values to this variables on block comment enter, restore them on exit */
+        let blockCommentDepth = 0;
```

Comment:
Let's add a formatting sample file with an indented block of code here

Reply:
Done.